pyproject.toml

```diff
@@ -403,6 +410,19 @@ mypy-init-return = true
 [tool.ruff.pydocstyle]
 convention = "numpy"
 
+[tool.repo-review]
+ignore = [
+    "MY",    # ignore MyPy setting checks
+    "PC111", # ignore using `blacken-docs` in pre-commit
```

Comment:
Though we should fix this eventually...

Reply:
The blacken docs, I mean.

Reply:
This _might_ (not at all guaranteed) come to Ruff one day. It's at least an open suggestion at https://github.com/astral-sh/ruff/issues/8237 and https://github.com/astral-sh/ruff/issues/3792. :)